data/SAbDab_Chen/meta.yaml

```diff
@@ -0,0 +1,50 @@
+    \ antibodies\n        that satisfy the following criteria: 1. \n        have both sequence (FASTA) and Protein Data Bank (PDB) structure files,\n  \
+    \      2. contain both a heavy chain and a light chain, and 3. \n        have crystal structures with resolution < 3 Å. \n        The DI label is derived\
+    \ from BIOVIA's pipelines."
+targets:
+    - id: developability
+      description: functional antibody candidate to be developed into a manufacturable(1), or not(0)
+      units: ''
+      type: categorical
+      names:
+          - antibody developability
+          - monoclonal anitbody
+          - functional antibody candidate
+          - manufacturable, stable, safe, and effective antibody drug
+      uris:
+          - https://rb.gy/idkdqp
+          - https://rb.gy/b8cx8i
+identifiers:
+    - id: antibody_pdb_ID
+      type: Other
+      description: anitbody pdb id
+    - id: heavy_chain
+      type: Other
+      description: anitbody heavy chain amino acid sequence in FASTA
+    - id: light_chain
+      type: Other
+      description: anitbody light chain amino acid sequence in FASTA
+license: CC BY 4.0
+links:
+    - url: https://doi.org/10.1101/2020.06.18.159798
+      description: corresponding publication
+    - url: https://doi.org/10.1093/nar/gkt1043
+      description: corresponding publication
+    - url: https://www.3ds.com/products-services/biovia/products/data-science/pipeline-pilot/
+      description: corresponding tools used
+    - url: https://tdcommons.ai/single_pred_tasks/develop/#sabdab-chen-et-al
+      description: data source
+num_points: 2409
+bibtex:
+    - "@article{Chen2020,\n          doi = {10.1101/2020.06.18.159798},\n          url = {https://doi.org/10.1101/2020.06.18.159798},\n          year =\
+      \ {2020},\n          month = jun,\n          publisher = {Cold Spring Harbor Laboratory},\n          author = {Xingyao Chen and Thomas Dougherty and\
+      \ \n          Chan Hong and Rachel Schibler and Yi Cong Zhao and \n          Reza Sadeghi and Naim Matasci and Yi-Chieh Wu and Ian Kerman},\n    \
+      \      title = {Predicting Antibody Developability from Sequence \n          using Machine Learning}}"
+    - "@article{Dunbar2013,\n          doi = {10.1093/nar/gkt1043},\n          url = {https://doi.org/10.1093/nar/gkt1043},\n          year = {2013},\n\
+      \          month = nov,\n          publisher = {Oxford University Press ({OUP})},\n          volume = {42},\n          number = {D1},\n          pages\
+      \ = {D1140--D1146},\n          author = {James Dunbar and Konrad Krawczyk and Jinwoo Leem \n          and Terry Baker and Angelika Fuchs and Guy Georges\
+      \ and Jiye Shi and\n          Charlotte M. Deane},\n          title = {{SAbDab}: the structural antibody database},\n          journal = {Nucleic\
+      \ Acids Research}}"
```

Comment:
I'm also surprised by the linebreaks here

Reply:
I guess this also due to the Å? Anyway, fixed!